sdks/python/scripts/run_pytest.sh

```diff
@@ -43,12 +48,7 @@ pytest -o junit_suite_name=${envname}_no_xdist \
   --junitxml=pytest_${envname}_no_xdist.xml -m 'no_xdist' ${pytest_args} --pyargs ${posargs}
 status2=$?
 
-# Exit with error if no tests were run in either suite (status code 5).
```

Comment:
Is removing this block intended?

Reply:
Yes (and I also intentionally slightly ammended the comment below) - since some environments may not have tests in every directory and we're now only running tests on certain directories at a given time, we need to be tolerant of empty pytest invocations.

The original motivating example was `dataframes` tests, but I think we can still run into this with other environments.

Reply:
To me like with current change there isn't a test suite hitting `$status1 == 5 && $status2 == 5` still, is that wrong? Still sounds not good to me to ignore not test running. I had experiences on running some local tests but did not feed the test with needed args and triggered `@ignore`. Currently I get warned by this clause.
If there is a test job that hitting this clause, one can fix that special case by sth like `sh run_pytest.sh ... || [[ $? == 5 ]] `to allow return code 5

Reply:
Yeah, that's fair. I'll see if I can add it back in

Reply:
Looks like this indeed worked, thanks for pushing on this